src/worker.cpp

```diff
@@ -860,11 +897,8 @@ void Worker::publishSystemVPD(const types::VPDMapVariant& parsedVpdMap)
 
     if (auto ipzVpdMap = std::get_if<types::IPZVpdMap>(&parsedVpdMap))
     {
+        (void)ipzVpdMap;
```

Comment:
Why this is needed here?

Reply:
for ipzVpdMap variable I was getting 'unused variable' error when I compile locally using ninja on docker.
as ipzVpdMap variable is not being used inside if case, I have removed creating this variable.